config.go

```diff
@@ -220,6 +223,28 @@ type Config struct {
 	// reclaimed by one with a different address or port. By default, this is 0,
 	// meaning nodes cannot be reclaimed this way.
 	DeadNodeReclaimTime time.Duration
+
+	CidrsAllowed []net.IPNet
+	CidrsDenied  []net.IPNet
+}
+
+// ParseCIDRs return a possible empty list of all Network that have been parsed
+// In case of error, it returns succesfully parsed CIDRs and the last error found
+func ParseCIDRs(v []string) ([]net.IPNet, error) {
+	nets := make([]net.IPNet, 0)
+	var err error
+	if v == nil {
+		return nets, err
+	}
+	for _, p := range v {
+		_, net, err := net.ParseCIDR(strings.TrimSpace(p))
+		if err != nil {
+			err = fmt.Errorf("invalid cidr: %s", p)
```

Comment:
If this func is intended to parse user config wouldn't it be better to fail if any one of the cidrs is invalid instead of silently ignoring it (even if a warning is printed on startup it can be easily overlooked) ?

Reply:
might be, the user caller can chose it anyway

Reply:
@rboyer DONE